test/integration/services/storage/storage-manger.spec.ts

```diff
@@ -0,0 +1,341 @@
+import config from 'config'
+import chai from 'chai'
+import sinonChai from 'sinon-chai'
+import BigNumber from 'bignumber.js'
+import { asciiToHex, hexToAscii } from 'web3-utils'
+
+import { getFeatherClient, TestingApp, ZERO_ADDRESS } from '../../utils'
+import Offer from '../../../../src/services/storage/models/offer.model'
+import BillingPlan from '../../../../src/services/storage/models/billing-plan.model'
+import Rate from '../../../../src/services/rates/rates.model'
+import Agreement from '../../../../src/services/storage/models/agreement.model'
+import StakeModel from '../../../../src/services/storage/models/stake.model'
+import { ServiceAddresses } from '../../../../src/definitions'
+import { WEI } from '../../../../src/services/storage/utils'
+
+chai.use(sinonChai)
+const expect = chai.expect
+
+const generateMsg = (peerId = 'FakePeerId') => {
```

Comment:
Hmm I am afraid this won't work. The `message` is `byte32[]` what you are returning is not an array and also is not `byte32` but `byte33`...

I have already solved this in Pinner, check it out here: https://github.com/rsksmart/rif-storage-pinner/blob/6db9a7eb092547738f671a8137fdaac835a5a211/test/utils.ts#L338

Reply:
Hmmm I see, you are not really passing real looking PeerId to it, so that might be the reason it works. I would still use the solution here, so we prevent some future problems here...

Reply:
Done